src/common/common.cpp

```diff
@@ -385,6 +385,25 @@ std::wstring get_resource_string(UINT resource_id, HINSTANCE instance, const wch
   }
 }
 
+// function to return the string as a to a wchar_t* (to enable localization in cases where the return type did not accept wstring)
+wchar_t* get_resource_string_wchar(UINT resource_id, HINSTANCE instance)
```

Comment:
This function and the corresponding `GET_RES_STRING_WCHAR` macro is only used in a couple occurrences, and both could be easily changed to work with `std::wstring`. 

Could you please remove these new functions and use the existing `get_resource_string`/`GET_RESOURCE_STRING` instead?